code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -95,7 +95,7 @@
 					user.add_mob_blood(src)
 					if(ishuman(user))
 						var/mob/living/carbon/human/dirtyboy = user
-						dirtyboy.adjust_hygiene(-10)
+						dirtyboy.adjust_hygiene(-4)

```

Comment:
is this and the other -4 suppose to be "linked"? If it's an amount that should always be the same for both prob should define.